scripts/Tools/code_checker

```diff
@@ -57,10 +57,16 @@ def run_pylint(on_file):
 ###############################################################################
     pylint = find_executable("pylint")
 
-    cmd = "%s --disable=I,C,R,logging-not-lazy,wildcard-import,unused-wildcard-import,fixme,broad-except,bare-except,eval-used,exec-used,global-statement %s" % (pylint, on_file)
+    cmd_options = " --disable=I,C,R,logging-not-lazy,wildcard-import,unused-wildcard-import,fixme,broad-except,bare-except,eval-used,exec-used,global-statement"
+
+    if "scripts/Tools" in on_file:
+        cmd_options +=",relative-import"
+    cmd = "%s %s %s"%(pylint, cmd_options, on_file)
+    print "HERE %s"%cmd
     stat, out, err = run_cmd(cmd, verbose=False)
     if stat != 0:
         logger.info("File %s has pylint problems, please fix\n    Use command: %s" % (on_file, cmd))
+        logger.info(out + "\n" + err)
```

Comment:
This is much lower impact now that we are doing one file at a time, removing lines below.